docs/qa/CometBFT-QA-34.md

```diff
@@ -87,26 +89,26 @@ We refer to these UUID to indicate to the representative runs.
 
 ### CometBFT Homogeneous network
 
-![latencies](./img/homogeneous/all_experiments.png)
+![latencies](img34/homogeneous/all_experiments.png)
```

Comment:
Why do these images need to be moved? Aren't they already under the `v034` directory?

Reply:
they were inside a `/v034/img` folder but we need to bring them one level up so that's why using `/img34` to avoid conflicts with `/v037/img` images